client/src/components/Container/index.js

```diff
@@ -21,6 +21,7 @@ function ContainerComponent({
   itemColumns,
   spacing,
   gridUserWidth,
+  displayNone,
```

Comment:
A better name for this prop might be `hidden`, to avoid being linked so rigidly to the actual style being applied if it's `true`.

In the future, if you changed the styles below to hide it in a different way (for example `visible: "hidden"` or `opacity: 0)` and your prop was still named `displayNone`, then it'd look wrong (and so you'd probably end up changing its name in this file and other files).